snuba/query/query_settings.py

```diff
@@ -91,6 +101,9 @@ def get_consistent(self) -> bool:
     def get_debug(self) -> bool:
         return self.__debug
 
+    def get_parent_api(self) -> str:
```

Comment:
why does this need to be in the query settings?

Reply:
I needed it in order to get some tests passing, but I think I was in a "Friday afternoon" haze and I did something wrong. I'm changing it.